src/main/java/com/qaprosoft/carina/demo/gui/pages/AutomationPracticeContactPage.java

```diff
@@ -0,0 +1,51 @@
+package com.qaprosoft.carina.demo.gui.pages;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class AutomationPracticeContactPage extends AbstractPage {
+
+    @FindBy(id = "id_contact")
+    private ExtendedWebElement subjectHeading;
+
+    @FindBy(id = "email")
+    private ExtendedWebElement emailAddress;
+
+    @FindBy(id = "id_order")
+    private ExtendedWebElement orderReference;
+
+    @FindBy(id = "message")
+    private ExtendedWebElement message;
+
+    @FindBy(id = "submitMessage")
+    private ExtendedWebElement send;
```

Comment:
the same as in comment above, let's name it `sendButton`
Please, fix it for all elements